apps/code-examples/src/app/code-examples/indicators/alert/basic/alert-demo.component.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { TestBed } from '@angular/core/testing';
+import { SkyAlertHarness } from '@skyux/indicators/testing';
+
+import { AlertDemoComponent } from './alert-demo.component';
+import { AlertDemoModule } from './alert-demo.module';
+
+describe('Basic alert', () => {
+  async function setupTest(days: number) {
```

Comment:
I would create an `options` parameter that includes the `days` as a property. That way, we can easily add more setup instructions in the future (this also demonstrates good practice for our consumers).
```
async function setupTest(options: { days: number; })

await setupTest({ days: 9 });
```

Reply:
I did this and made the options optional.